### PR TITLE
DOCS-2848: 13.0-RC1 Release Notes

### PR DESCRIPTION
**Pull request by jervin777**
Initial Release Notes Draft.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/ReleaseNotes/CORE/13.0RC1.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+	  </tr>
+	  <tr>
+	    <td><a href="https://jira.ixsystems.com/browse/NAS-114595" target="_blank">NAS-114595</a></td>
+	    <td>VNC can't connect to bhyve VMs.</td>
+	    <td>Update to 13.0 Nightlies or RC.1 (when released).</td>
+	  </tr>
+	  <tr>
+            <td></td>
+            <td>TrueNAS 12 can not replicate to or from TrueNAS 13</td>
+            <td>By default, TrueNAS 12 cannot initiate a replication to or from TrueNAS 13 due to an outdated SSH client library.  Allowing replication to or from TrueNAS 13 to TrueNAS 12 requires allowing ssh.rsa algorithms. See <a href="https://www.openssh.com/txt/release-8.2" target="_blank">OpenSSH 8.2 Release</a> for security considerations. Log into the TrueNAS 13 system and go to <b>Services->SSH</b>.  Add the <b>SSH Auxiliary Parameter</b>: <code>PubkeyAcceptedAlgorithms +ssh-rsa</code>.</td>
```
Comment (micjohnson777) [inline content/ReleaseNotes/CORE/13.0RC1.md:233]:
```suggestion
            <td>By default, TrueNAS 12 cannot initiate a replication to or from TrueNAS 13 due to an outdated SSH client library. Allowing replication to or from TrueNAS 13 to TrueNAS 12 requires allowing ssh.rsa algorithms. See <a href="https://www.openssh.com/txt/release-8.2" target="_blank">OpenSSH 8.2 Release</a> for security considerations. Log into the TrueNAS 13 system and go to <b>Services->SSH</b>. Add the <b>SSH Auxiliary Parameter</b>: <code>PubkeyAcceptedAlgorithms +ssh-rsa</code>.</td>
```

**Review by micjohnson777 — COMMENTED** [review]
In the lists of tickets, the descriptions should begin with capital letters

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
This is approved for merge - link check is hitting some intermittent 500 errors after resolving previous failures.

**Review by ericbsd — DISMISSED** [review]
LGTM

**Pull request merged**